ecl/hql/hqlgram2.cpp

```diff
@@ -12379,15 +12379,18 @@ IHqlExpression * PseudoPatternScope::lookupSymbol(IIdAtom * name, unsigned looku
 
 
 //---------------------------------------------------------------------------------------------------------------------
-
 extern HQL_API IHqlExpression * parseQuery(IHqlScope *scope, IFileContents * contents, HqlLookupContext & ctx, IXmlScope *xmlScope, IProperties * macroParams, bool loadImplicit, bool isRoot)
```

Comment:
I would have probably added to 
    parseQuery(const char * text, IErrorReceiver * errs)
since it would have a smaller effect, but this should also be fine.